packages/scaffolding-cli/src/domain/config/questions.ts

```diff
@@ -81,13 +81,13 @@ export function cliQuestions(default_project_name: string): Array<PromptQuestion
             // "type": (prev: boolean) => prev ? "confirm" : "",
             "type": "confirm",
             "name": "create_config",
-            "message": "Create a sample JSON config with all selected values for future runs of the CLI?",
+            "message": "Create a sample JSON config (with all selected values for future runs of the CLI)?",
             "initial": true
         },
         {
             "type": "confirm",
             "name": "advanced_config",
-            "message": "Enable Advanced Config? Specifying Cloud/Terraform/Business details",
+            "message": "Enable Advanced Config (specifying cloud/Terraform/business details)?",
```

Comment:
What do you think would make sense to a user? `Advanced config`? Or maybe `continue to configure advanced scaffolding options, including cloud infra setup, and further project business details`?

Reply:
I like "Continue to Advanced Config...."